React/Modules/RCTDevMenu.h

```diff
@@ -58,6 +58,26 @@
 - (void)reload;
 
 /**
+ * Manually enable or disable remote JS debugging (can be called from JS)
```

Comment:
You don't need to change this header if you're just using this for RCT_EXPORT_METHOD.